extensions/packageManager/src/client/Library.tsx

```diff
@@ -7,6 +7,7 @@ import React, { useState, Fragment, useEffect } from 'react';
 import formatMessage from 'format-message';
 import {
   Link,
+  PrimaryButton,
```

Comment:
It's better if you import these components as modules from their own package, for examples:
```import { PrimaryButton } from from 'office-ui-fabric-react/lib/Button';```

Reply:
@hatpick in the case of extensions, the Fabric lib is attached to the window so it is excluded from the bundle.